src/standard/utils.html

```diff
@@ -306,6 +306,17 @@
         }
       }
       return elt;
+    },
+
+    /**
+     * Checks whether an element is in this element's shadow DOM tree.
+     *
+     * @method isShadowDomChild
+     * @param {HTMLElement=} node The element to be checked.
+     * @return {Boolean} true if node is in this element's shadow DOM tree.
+     */
+    isShadowDomChild: function(node) {
```

Comment:
The name should be `isLocalDomChild`. Shadow and Shady are both implementations Polymer uses to provide local DOM.